spec/reek/smells/smell_warning_spec.rb

```diff
@@ -96,12 +96,22 @@
     end
 
     it 'matches on class symbol' do
-      expect(uncommunicative.matches?(:UncommunicativeVariableName)).to be_truthy
+      expect(uncommunicative.matches?(:UncommunicativeVariableName)).to\
+        be_truthy
     end
 
     it 'matches on class symbol and params' do
       expect(uncommunicative.matches?(:UncommunicativeVariableName,
-                                      test: 'something')).to be_truthy
+                                      parameters: {
+                                        test: 'something'
+                                      })).to be_truthy
+    end
+
+    it 'matches on class symbol, params and vars' do
```

Comment:
"vars" is not a good name here and below (highly ambiguous). We introduced the "attributes" terminology above and should stick with it.